src/connector.js

```diff
@@ -30,106 +37,50 @@ const defaults = {
 	"isAdvancedSearch": false,
 	"originLevel3": window.location.origin + winPath
 };
-const monthsEn = [ "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec" ];
-const monthsFr = [ "janv.", "févr.", "mars", "avr.", "mai", "juin", "juil.", "août", "sept.", "oct.", "nov.", "déc." ];
-
 let lang = document.querySelector( "html" )?.lang;
+let paramsOverride = baseElement ? JSON.parse( baseElement.dataset.gcSearch ) : {};
```

Comment:
Need a mechanism to prevent a JS error when the data attribute `data-gc-search` is undefined.

In such situation, the code execution should be abort to prevent interference and/or conflict.

Reply:
@duboisp I had thought the same initially, but then I changed my mind and decided to go with this code since `baseElement` is created from query selector `[data-gc-search]`, so it is an impossible scenario.

Reply:
Yes, I mist the code which define the `baseElement` at line 19 and the conditional start-up check at line 97. So it's all good.